.github/workflows/coverage.yml

```diff
@@ -0,0 +1,24 @@
+on: [pull_request]
+name: Code Coverage
+jobs:
+  coverage:
+    runs-on: "ubuntu-latest"
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Calc coverage
+      run: |
+        export PATH=$PATH:$(go env GOPATH)/bin
```

Comment:
Is this really necessary? If `go` is not in `PATH`, `go env` would also fail.

Reply:
You are right; I just copied what was present before; I will remove this.
thank you